tools/iceoryx_build_test.sh

```diff
@@ -49,6 +50,11 @@ do
         "strict")
             STRICT_FLAG="on"
             ;;
+        "qacpp")
+            BUILD_TYPE="Release"
+            QACPP_JSON="on"
+            INTROSPECTION_FLAG="off"
```

Comment:
Please enable introspection, we want to scan it too.

Reply:
Done